circle.yml

```diff
@@ -505,6 +545,12 @@ workflows:
           filters:
             tags:
               only: /.*/
+      - release-windows:
+          filters:
+            branches:
+              ignore: /.*/
```

Comment:
Branches should not be ignored. You can also enabled builds of release-macos.

Reply:
OK.